OrbitGl/CaptureWindow.cpp

```diff
@@ -1077,29 +896,4 @@ void CaptureWindow::RenderTimeBar() {
 }
 
 //-----------------------------------------------------------------------------
-void CaptureWindow::Initialize() {
-  GlCanvas::Initialize();
-  LoadIcons();
-}
-
-//-----------------------------------------------------------------------------
-uint32_t LoadIcon(const char* name) {
-  std::string icon_path = Path::GetExecutablePath() + "icons/" + name;
-  return LoadTextureFromFile(icon_path.c_str());
```

Comment:
I thought about removing it, but I think it could be useful in the future. Actually let me remove it in another PR, we can alway revert the removal commit if we need it again.

Reply:
Sounds good!